<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_LayerTreeOptionsDialog</class>
 <widget class="QDialog" name="LC_LayerTreeOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>268</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Layers Tree Options</string>
  </property>
  <layout class="QFormLayout" name="formLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabNaming">
      <property name="toolTip">
       <string>Settings for naming convention that is used to create layers hierarchy based on flat list of layers</string>
      </property>
      <attribute name="title">
       <string>Naming Convention</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="leLayerLevelSeparator">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Separator used for split inner name of layer to hierarchical names</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Layer Level Name Separator</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Secondary Layer Suffixes</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Dimensional</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leDimSuffix">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Suffix of layer's name that defines that layer is dimensional</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Informational</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leInfoSuffix">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Suffix of layer's name that defines that layer is informational.</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Alternative Position </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leAltPosSuffix">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Suffix of layer's name that defines that layer is for alternative position.</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Duplicating Name</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Prefix</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leDuplicatedPrefix">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Prefix of layer's name that is used if layer with such name exists (say, when duplicate is created)</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_61">
            <property name="text">
             <string>Suffix</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leDuplicatedSuffix">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Suffix of layer's name that is used if layer with such name exists (say, when duplicate is created)</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDisplay">
      <property name="toolTip">
       <string>Display and operations settings</string>
      </property>
      <attribute name="title">
       <string>Display and Operations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbShowToolTip">
         <property name="toolTip">
          <string>If checked, tooltip will include complete name of layer (with all upper layer levels)</string>
         </property>
         <property name="text">
          <string>Show full name as ToolTip</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbShowTypeIcons">
         <property name="toolTip">
          <string>If checked, icons for layer types will be shown</string>
         </property>
         <property name="text">
          <string>Show layer type icons</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="cbShowIdented">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Indentation for child layers</string>
         </property>
         <property name="text">
          <string>Show indented layer names. Ident size:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="sbIndentSize">
         <property name="suffix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>120</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="cbDragDrop">
         <property name="toolTip">
          <string>If checked, drag&amp;drop operations in layers list are enabled and layers hierarchy may be restructutured</string>
         </property>
         <property name="text">
          <string>Allow restructuring via Drag&amp;Drop</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="cbRenameWithPrimary">
         <property name="toolTip">
          <string>If primary layer is renames, secondary layers are renamed too.</string>
         </property>
         <property name="text">
          <string>Rename secondary layers on primary rename</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColors">
      <property name="toolTip">
       <string>Various colors settings</string>
      </property>
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QComboBox" name="cbSelectedItemBgColor">
         <property name="toolTip">
          <string>Background color for selected items in layers tree.</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbGridColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Color for separator lines in layers tree view</string>
         </property>
         <property name="accessibleDescription">
          <string notr="true"/>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string notr="true">#000000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">#FFFFFF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">#F0F8FF</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbVirtualLayerBackgroundColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Background color for virtual layers items in layers list</string>
         </property>
         <property name="accessibleDescription">
          <string notr="true"/>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string notr="true">#1E90FF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">#FFC200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pb_selectedItemColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Color selection</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/ui/ui.qrc">
           <normaloff>:/ui/colorxx.png</normaloff>:/ui/colorxx.png</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Matched item in highlight filter mode</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cbActiveLayerBgColor">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbHighlightedColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Color for items that matches filter conditions</string>
         </property>
         <property name="accessibleDescription">
          <string notr="true"/>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string notr="true">#FFC200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>#1E90FF</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pb_highlightedColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Color selection</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/ui/ui.qrc">
           <normaloff>:/ui/colorxx.png</normaloff>:/ui/colorxx.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lRelativeZeroColor">
         <property name="text">
          <string>Virtual layer background color</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pb_gridColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Color selection</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/ui/ui.qrc">
           <normaloff>:/ui/colorxx.png</normaloff>:/ui/colorxx.png</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lBackground">
         <property name="text">
          <string>Threeview items separator grid</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Selected items background color</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pbActiveLayerBgColor">
         <property name="toolTip">
          <string>Color selection</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/ui/ui.qrc">
           <normaloff>:/ui/colorxx.png</normaloff>:/ui/colorxx.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pbSelectedItemsBgColor">
         <property name="toolTip">
          <string>Color selection</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/ui/ui.qrc">
           <normaloff>:/ui/colorxx.png</normaloff>:/ui/colorxx.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="toolTip">
          <string>Background color for active layer item in layers tree</string>
         </property>
         <property name="text">
          <string>Active layer background color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPens">
      <property name="toolTip">
       <string>Pens that will be set for newly created layers of different types.</string>
      </property>
      <attribute name="title">
       <string>Default Pens</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QG_WidgetPen" name="wPenDimensional" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="QG_WidgetPen" name="wPenInfo" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="QG_WidgetPen" name="wPenNormal" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QG_WidgetPen" name="wPenAltPos" native="true"/>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QG_WidgetPen</class>
   <extends>QWidget</extends>
   <header>qg_widgetpen.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../res/ui/ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LC_LayerTreeOptionsDialog</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LC_LayerTreeOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_highlightedColor</sender>
   <signal>clicked()</signal>
   <receiver>LC_LayerTreeOptionsDialog</receiver>
   <slot>pb_highlightedColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_gridColor</sender>
   <signal>clicked()</signal>
   <receiver>LC_LayerTreeOptionsDialog</receiver>
   <slot>pb_gridColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSelectedItemsBgColor</sender>
   <signal>clicked()</signal>
   <receiver>LC_LayerTreeOptionsDialog</receiver>
   <slot>pbSelectedItemsBgColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbActiveLayerBgColor</sender>
   <signal>clicked()</signal>
   <receiver>LC_LayerTreeOptionsDialog</receiver>
   <slot>pbActiveLayerBgColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_selectedItemColor</sender>
   <signal>clicked()</signal>
   <receiver>LC_LayerTreeOptionsDialog</receiver>
   <slot>pb_selectedItemColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowIdented</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LC_LayerTreeOptionsDialog</receiver>
   <slot>showIndentedClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validate()</slot>
  <slot>pb_highlightedColorClicked()</slot>
  <slot>pb_gridColorClicked()</slot>
  <slot>pbSelectedItemsBgColorClicked()</slot>
  <slot>pbActiveLayerBgColorClicked()</slot>
  <slot>pb_selectedItemColorClicked()</slot>
  <slot>showIndentedClicked()</slot>
 </slots>
</ui>
